mmeval/core/dist_backends/oneflow_dist.py

```diff
@@ -0,0 +1,138 @@
+                              ' please install oneflow first.')
+
+    @property
+    def is_initialized(self) -> bool:
+        """Returns True if the distributed environment has been initialized.
+
+        Returns:
+            bool: Returns True if the distributed environment has been
+            initialized, otherwise returns False.
+        """
+        try:
+            flow.env.get_world_size()
+            is_init = True
+        except ValueError:
+            is_init = False
+        return is_init
+
+    @property
+    def rank(self) -> int:
+        """Returns the rank index of the current process group."""
+        return flow.env.get_local_rank()
+
+    @property
+    def world_size(self) -> int:
+        """Returns the world size of the current process group."""
+        return flow.env.get_world_size()
+
+    def _object_to_tensor(self, obj: Any) -> Tuple[Tensor, Tensor]:
+        """Convert the given object to a tensor via `pickle.dumps`.
+
+        Args:
+            obj (any): Any pickle-able python object.
+
+        Returns:
+            Tuple: A tuple of the tensor converted from given object and the
+            tensor size.
+        """
+        device = 'cuda'
+        if flow.cuda.device_count() == 0:
+            device = 'cpu'
+        buffer = pickle.dumps(obj)
+        storage = np.frombuffer(buffer, dtype=np.int8)
+        tensor = flow.tensor(storage).to(device)
+        local_size_tensor = flow.tensor([tensor.numel()], device=device)
+        return tensor, local_size_tensor
+
+    def _tensor_to_object(self, tensor: Tensor,
+                          tensor_size: Union[int, Tensor]) -> Any:
+        """Convert the given Tensor to a object via `pickle.loads`.
+
+        Args:
+            tenosr (Tensor): A tensor-like data.
+            tensor_size (int or Tensor): The tensor size of the given Tensor to
+                be convert object.
+
+        Returns:
+            Any: The object converted from the given tensor.
+        """
+        size = int(tensor_size)
+        buffer = tensor.cpu().numpy().tobytes()[:size]
+        obj = pickle.loads(buffer)
+        return obj
+
+    def _pad_tensor(self, tensor: Tensor,
+                    max_size: Union[int, Tensor]) -> Tensor:  # yapf: disable
+        """Padding the given tensor to the given size.
+
+        Args:
+            tensor (Tensor): A tensor-like data to be padded.
+            max_size (int or Tensor): The max tensor size that for tensor
+                padding.
+
+        Returns:
+            Tensor: The padded tensor.
+        """
+        if tensor.numel() < max_size:
+            print(f'max_size{max_size}, tensor:{tensor}')
+            padding = flow.zeros((max_size - tensor.numel(), ),
+                                 dtype=flow.int8,
+                                 device=tensor.device)
+            tensor = flow.cat((tensor, padding), dim=0)
+        return tensor
```

Comment:
I think we can remove the check statement:
```
if tensor.numel() < max_size:
```

If `tensor.numel()` is equal to `max_size`, we got a padding tensor with size 0, which also works fine.

Reply:
get